tests/queryset_pickle/tests.py

```diff
@@ -66,6 +66,20 @@ def test_model_pickle(self):
         self.assertEqual(original, reloaded)
         self.assertEqual(original.somefield, reloaded.somefield)
 
+    def test_model_pickle_with_ignoring_fields(self):
+        """
+        Test that developer can choose what fields he needs to pickle
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Tests that" since all tests test things. I would write something like `Model.__getstate__() allows choosing which attributes to pickle.`